BWAPI4J/src/main/java/org/openbw/bwapi4j/BW.java

```diff
@@ -331,11 +330,6 @@ private void loadLibraries(final List<String> libraries) {
 
     return libNames;
   }
-
-  private static String getLibraryPathDelimiter() {
```

Comment:
We are now at the mercy of the JDK/JRE deciding what the proper delimiter is. But, I am willing to try `File.pathSeparator` out.

Reply:
Have there been instances where the JVM is detecting the wrong system? If so I would just consolidate the methods since the logic to determine the delimiter is duplicated.

Reply:
No instances that I know of nor for which I have any evidence. As I said, we can try it out no big deal and it's probably fine. I just like having more control over simple things rather than relying on a library (even the JDK) to probe the system for something simple.